prime-router/src/main/kotlin/azure/SendFunction.kt

```diff
@@ -160,8 +173,8 @@ class SendFunction(private val workflowEngine: WorkflowEngine = WorkflowEngine()
                 ReportEvent(Event.EventAction.SEND_ERROR, reportId, isEmptyBatch)
             } else {
                 // retry using a back-off strategy
-                val waitMinutes = retryDuration.getOrDefault(nextRetryCount, maxDurationValue)
-                val randomSeconds = Random.nextInt(-30, 31)
```

Comment:
Moved into constant since -30 could produce negative time if first retry is < 30s.